libs/community/langchain_community/document_compressors/flashrank_rerank.py

```diff
@@ -69,11 +71,10 @@ def compress_documents(
         final_results = []
 
         for r in rerank_response:
-            metadata = r["meta"]
-            metadata["relevance_score"] = r["score"]
-            doc = Document(
-                page_content=r["text"],
-                metadata=metadata,
-            )
-            final_results.append(doc)
+            if r["score"] > self.score_threshold:
```

Comment:
In the doc string it's indicated that it's the "Minimum relevance threshold to return", this mean that document with this exact score should be included in the result.

```suggestion
            if r["score"] >= self.score_threshold:
```